### ISSUE TITLE
JAWS 2021 not reading element when focus is moved to it using Javascript

### ISSUE THREAD

**Issue by markmagennis** [FreedomScientific/VFO-standards-support#564]
### Summary

When a Javascript click event handler is used to move focus to a div with text content, JAWS 2021 does not read the text content. This is different behavior to JAWS 2020 and not welcome.

If focus is moved on page load using window.onload = function() {document.getElementById("xxx").focus();}; then JAWS reads the contents of the div. But if focus is moved as a result of a button press using <button onclick=document.getElementById("xxx").focus();</button> then JAWS 2021 doesn't read the text content even though JAWS 2020 does.

I believe it is good that an element is read whenever focus is moved to it so I would like to see the JAWS 2020 functionality reinstated in JAWS 2021.

Steps to reproduce:
1. Run JAWS 2021
2. Open https://b0w6v.csb.app/focus_move_test.html
3. Tab to one of the buttons and press Enter
4. Tab to the other button and press Enter
5. Repeat with JAWS 2020

### Expected result

When the page loads and focus is moved to the results statement, JAWS reads the results statement.
When either button is pressed and focus is moved to the results statement, JAWS reads the results statement.

### Actual result

JAWS 2020 reads the results statement each time focus is moved to it - on page load and when a button is pressed.
JAWS 2021 only reads it on page load, but not when focus is moved by a button press.

### Example

https://b0w6v.csb.app/focus_move_test.html

#### JAWS version and build number

2021.2107.12 ILM
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 92.0.4515.131
